Misc/NEWS.d/next/Library/2024-10-31-14-45-52.gh-issue-101123.XRpEJs.rst

```diff
@@ -0,0 +1 @@
+Fix :func:`math.hypot` signature.
```

Comment:
IMO, we can drop the NEWS entry.

Reply:
This is somewhat a user-visible change.

Reply:
Yes, but [we normally do not produce NEWS entries for doc improvements](https://devguide.python.org/core-developers/committing/#changes-that-require-news-entries).

Reply:
If you would like the NEWS entry, consider instead to put it in the _Documentation_ category.

Moreover, if you'd want, you can also create another NEWS entry for the Argument Clinic adaptions; we sometimes produce those. See the Git log and decide what you think makes most sense for your change :)

Reply:
If you want to keep the NEWS entry as it is, give me a thumbs-up and I'll land this.

Reply:
Fair enough.